apps/web/src/components/layout/components/PrivatePageLayout.tsx

```diff
@@ -45,7 +47,22 @@ export function PrivatePageLayout() {
     [environment]
   );
 
-  return (
+  function clerkPrivateWrapper(children: React.ReactNode) {
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** I believe there are performance concerns with this since it will re-render _the entire app / component tree‏_ on every re-render. https://stackoverflow.com/a/59776934